docs/modules/integrate/pages/kafka-connect-connectors.adoc

```diff
@@ -117,4 +118,18 @@ xref:pipelines:submitting-jobs.adoc#submitting-a-job-using-a-java-client-or-embe
 
 == Fault Tolerance
 
-When a Kafka Connect Source connector is deployed to a Hazelcast cluster, snapshots of the connector state are stored in the Jet engine. For example, partition offsets and any metadata required for a restart or recovery. If there is a connector failure, the recovery behavior will depend on the type of connector. Refer to the documentation of your Kafka Connect Source connector for more detailed information. 
+When a Kafka Connect Source connector is deployed to a Hazelcast cluster, snapshots of the connector state are stored by the Jet engine.
+Example for such states are partition offsets and any metadata required for a restart or recovery. If there is a connector failure, the recovery behavior will depend on the type of connector. Refer to the documentation of your Kafka Connect Source connector for more detailed information.
+
+== Parallelism and reconfigurations
+
+Kafka Connect Source uses `tasks.max` configuration property to determine parallelism.
+Even if pipeline's total parallelism is higher than `tasks.max` value, the `tasks.max` value takes precedence.
+It is prohibited to have pipeline's total parallelism (edge's local parallelism * number of nodes) lower than `tasks.max`.
+
+After creation of an instance of a SourceConnector, Hazelcast Jet asks it to provide `tasks.max` task configurations. If the
```

Comment:
Hazelcast Jet requests the `task.max` task configuration when an instance of a Source Connector has been created, .